index.html

```diff
@@ -14,10 +14,40 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
+<body class="background-lightgray">
+  <header class="purple">
+    <div class = "container">
+      <div class="row column-full">
+        <h2 class="white-text">Code Journal</h2>
+      </div>
+    </div>
   </header>
   <main>
+    <div data-view="entry-form" class="container">
+      <form>
+        <div class="row column-full">
+          <h1>New Entry</h1>
+        </div>
+        <div class="row">
+          <div class = "row column-half cover">
```

Comment:
A single element should not be both a `row` and a `column`. If you need a `row` in a `column`, you should make a child element that is a `row` in the `column` element

Reply:
There are a couple other elements that are both `row` and `column` in your HTML, please revise those as well